config/projects/sensu.rb

```diff
@@ -89,6 +89,9 @@
 #  config_file "/etc/default/sensu"
 #end
 
+# Make sure Windows gets a gem.bat
+dependency "shebang-cleanup"
```

Comment:
Actually, this should probably be wrapped to only apply on Windows since I don't want to adversely affect other platforms until we have to.